### PR TITLE
Fixes to AWS `wrap-guest-image` command

### PR DESCRIPTION
**Pull request by krishnanrs**
- Added the missing verbose parameter, which was causing the
   command to fail
 - Print the `wrap-guest-image` in the top-level help of supported
   commands
 - Added a means to identify EBS optimized instances and pass the
   flag accordingly

### CODE REVIEW

File: brkt_cli/aws/aws_service.py [hunk 1]
```diff
@@ -28,6 +28,23 @@
 
 log = logging.getLogger(__name__)
 
+EBS_OPTIMIZED_INSTANCES = ['c1.xlarge', 'c3.xlarge', 'c3.2xlarge',
```
Comment (krishnanrs) [inline brkt_cli/aws/aws_service.py]: If there is any better suggestion for identifying EBS optimized instances (or to place the list of EBS optimized instances in a different place), let me know.

File: brkt_cli/aws/wrap_image.py [hunk 2]
```diff
@@ -134,13 +135,17 @@ def launch_wrapped_image(aws_svc, image_id, metavisor_ami,
...
+        if instance_type not in EBS_OPTIMIZED_INSTANCES:
+            ebs_optimized = False
         instance = aws_svc.run_instance(
             mv_image.id,
             instance_type=instance_type,
             security_group_ids=security_group_ids,
             user_data=compressed_user_data,
             placement=None,
             block_device_map=bdm,
+            ebs_optimized=ebs_optimized,
```
Comment (bburtin) [inline brkt_cli/aws/wrap_image.py]: `ebs_optimized=instance_type in EBS_OPTIMIZED_INSTANCES`

File: brkt_cli/aws/wrap_image_args.py [hunk 3]
```diff
@@ -41,6 +42,13 @@ def setup_wrap_image_args(parser, parsed_config):
     aws_args.add_subnet(parser)
     aws_args.add_aws_tag(parser)
     aws_args.add_key(parser)
+    parser.add_argument(
```
Comment (bburtin) [inline brkt_cli/aws/wrap_image_args.py]: Please add a comment, saying that this is deprecated and should be removed once we get rid of `-v` at the subcommand level.

Comment (krishnanrs) [inline brkt_cli/aws/wrap_image_args.py]: Done

**Pull request merged**